### PR TITLE
chore(cloudfront): update README.md with examples on how to use pre-existing lambda@edge functions with cloudfront

### PR DESCRIPTION
**Pull request by robertd**
Relates to https://github.com/aws/aws-cdk/issues/9901

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/@aws-cdk/aws-lambda/README.md [hunk 1]
```diff
@@ -423,6 +423,27 @@ new lambda.Function(this, 'Function', {
 });
 ```
 
+### Existing Lambda function with CloudFront distribution
```
Comment (nija-at) [inline packages/@aws-cdk/aws-lambda/README.md:426]: It doesn't feel like this belongs in lambda's README. 

Lambda@Edge is a cloudfront feature and and seems to fit nicely there. Can you revert this file?

**Review by nija-at — CHANGES_REQUESTED** [review]

File: packages/@aws-cdk/aws-lambda/README.md (hunk 1 above)
Comment (robertd) [reply packages/@aws-cdk/aws-lambda/README.md]: Done.

File: packages/@aws-cdk/aws-cloudfront/README.md [hunk 2]
```diff
@@ -225,6 +225,25 @@ myDistribution.addBehavior('images/*', myOrigin, {
...
+  defaultBehavior: {
+    origin: new origins.S3Origin(s3Bucket),
+    edgeLambdas: [
+       {
+         functionVersion,
+         eventType: cloudfront.LambdaEdgeEventType.VIEWER_REQUEST
+       },
+    ],
+  },
+  ...
```
Comment (njlynch) [inline packages/@aws-cdk/aws-cloudfront/README.md:243]: I'd rather just remove this, as the code is perfectly valid without, and not having uncompilable bits will help with us translating this across languages.

```suggestion
```

File: packages/@aws-cdk/aws-cloudfront/README.md [hunk 3]
```diff
@@ -225,6 +225,25 @@ myDistribution.addBehavior('images/*', myOrigin, {
 });
 ```
 
+Adding an existing Lambda@Edge function created in a different stack to a CloudFront distribution.
+
+```ts
+const functionVersion = lambda.Version.fromVersionArn(this, 'Version', 'arn:aws:lambda:us-east-1:123456789012:function:functionName:1');
+
+const distro = new cloudfront.Distribution(this, "distro", {
```
Comment (njlynch) [inline packages/@aws-cdk/aws-cloudfront/README.md:233]:
```suggestion
new cloudfront.Distribution(this, 'distro', {
```

**Review by njlynch — COMMENTED** [review]
Looks great. Two minor nitpicks.

**Review by njlynch — DISMISSED** [review]

**Review by njlynch — APPROVED** [review]

**Pull request merged**